types/react-router-redux/index.d.ts

```diff
@@ -25,15 +25,15 @@ export interface ConnectedRouterProps<State> {
 }
 export class ConnectedRouter<State> extends React.Component<ConnectedRouterProps<State>> {}
 
-export const LOCATION_CHANGE: string;
+export const LOCATION_CHANGE: '@@router/LOCATION_CHANGE';
```

Comment:
@dborysov Only when there is assignment of type const x = "somestring" is when compiler would infer it as string literal type instead of ```string``` Hence you can write that declaration as ```=```